spec/features/unauth_user_cant_see_admin_dashboard_spec.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.feature "Unauthenticated user tries to go to admin dashboard" do
+  scenario "gets a 404 error" do
+    visit admin_dashboard_path
+
+    expect(page).to have_content("404")
+  end
+end
```

Comment:
I think this is the shortest test ever. I like it!

Reply:
haha ya i didn't know if I need much more. I thought about putting it in another file but realized we can start putting test files into folders (admin, authenticated user, unauthenticated user) just to clean it up and find all the tests a bit eastier.